gnocchiclient/v1/metric.py

```diff
@@ -38,11 +39,22 @@ def list(self, limit=None, marker=None, sorts=None):
         :param sorts: list of resource attributes to order by. (example
                       ["user_id:desc-nullslast", "project_id:asc"]
         :type sorts: list of str
-
+        :type follow_links_next: Whether to follow the next link (pagination)
+                                 even if limit is set.
```

Comment:
can you provide a bit more details on what this option does? it seems strange and seems like the user has to know implementation details to understand what it does.

Reply:
thinking about it, I think it's useless. updated the patch.